### PR TITLE
Send schedule using web socket

### LINKED ISSUE #113: Send schedule through web socket
### Feature you want to implement
As a part of #112, I will send schedule to each client through web socket.

### How the feature is implemented
Using a web socket, each client only need to connect to the proxy server once.
The server sends the updated schedule whenever it is modified, and clients always wait until a packet is received.

### PR DESCRIPTION
**Pull request by BECATRUE**
Now, a client i.e. IQUIP can fetch schedule using a web socket.

For test, run the code below.
```python
import json
from websockets.sync.client import connect

def main():
    uri = "ws://localhost:8000/schedule/"

    with connect(uri) as websocket:
        while True:
            response = websocket.recv()
            data = json.loads(response)
            print(data)

main()
```

Its result is the following.

image

Additionally, I found out we didn't check Pylint for other files except for `main.py`, thus I updated it.

This closes #113.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -205,26 +206,28 @@ async def get_experiment_info(file: str) -> Any:
...
+    await websocket.accept()
     try:
-        await asyncio.wait_for(schedule_tracker.modifed.wait(), timeout)
-    except asyncio.TimeoutError:
-        return latest, schedule
-    return schedule_tracker.get()
+        schedule = schedule_tracker.get()
+        await websocket.send_json(schedule)
+        while True:
+            await asyncio.wait_for(schedule_tracker.modifed.wait(), None)
```
Comment (kangz12345) [inline main.py]: Isn't it equivalent to:
```python
await schedule_tracker.modified.wait()
```

**Review by kangz12345 — CHANGES_REQUESTED** [review]
`WebSocket` looks so cool!

File: main.py (hunk 1 above)
Comment (BECATRUE) [reply main.py]: Wow... Fantastic😮

**Review by kangz12345 — APPROVED** [review]
LGTM

**Pull request merged**